### DISCUSSION TITLE
Distributions & fillers

### DISCUSSION THREAD

**Discussion by userban3000** [EwyBoy/OreTweaker#87] [Help Wanted!] [answered]
Is there a list of all available distribution and filler settings? I cannot seem to find such a thing anywhere in the wiki.

**Comment by EwyBoy** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/5883716/201211236-d95e4690-832d-4aec-aa21-2a5eb4a87a87.png)
Distribution can either be set to TRIANGLE or UNIFORM. Look at the image above for reference.
Triangle is these sideways pyramids. (More likly to spawn in the center between your minY and maxY)
Uniform is the square towers. (Uniform distribution between minY and maxY)

All vanilla Minecraft tags that can be used in fillers are listed here:
https://minecraft.fandom.com/wiki/Tag

minecraft:stone_ore_replaceables
